examples/api/lib/material/refresh_indicator/refresh_indicator.2.dart

```diff
@@ -0,0 +1,92 @@
+import 'package:flutter/material.dart';
+
+/// Flutter code sample for [RefreshIndicator.noSpinner].
```

Comment:
You'll also need a test for the example. Put it in examples/api/test/material/refresh_indicator/refresh_indicator.2_test.dart

The test just needs to exercise the example enough to know that the basics are working.